Remove inadvertently-committed file

DaveCTurner opened the pull request:
Introduced in #68047

DaveCTurner committed:
Remove inadvertently-committed file

Introduced in #68047

elasticmachine commented:
Pinging @elastic/es-core-infra (Team:Core/Infra)

pgomulka reviewed (approved):
no idea how did I commit this.. 
thank you for spotting

DaveCTurner commented:
:grin: I only noticed it because I also accidentally committed a bunch of `*.java.new` files on a branch somehow, enough that it was noticeable, so I deleted them all with `find` which also deleted this one.

DaveCTurner committed:
Merge branch 'master' into 2021-02-15-remove-unneeded-file
